File: NetFramework/DashboardLayout/CS/DashboardLayoutDemo/DemoControls/Label.cs
using C1.Win.C1Input;

namespace DashboardDemo.Controls
{
    public class DemoLabel : C1Label, IDemo
    {
        public void Init()
        {
            DataSource = SampleHelper.Instance.GetSourceTable("Total Sales", Properties.Resources.TotalSales);
            DataField = "Sales";
        }
    }
}

File: NetFramework/DashboardLayout/VB/DashboardLayoutDemo/DemoControls/Label.vb
Imports C1.Win.C1Input

Namespace DashboardDemo.Controls
    Public Class DemoLabel
        Inherits C1Label
        Implements IDemo

        Public Sub Init() Implements IDemo.Init
            DataSource = SampleHelper.Instance.GetSourceTable("Total Sales", Resources.TotalSales)
            DataField = "Sales"
        End Sub
    End Class
End Namespace
